### ISSUE TITLE
Custom icon images in home icon nav are missing alt attribute

### ISSUE THREAD

**Issue by cjg89** [UCF/Coronavirus-Child-Theme#37]
**Describe the bug**

When a custom icon image is uploaded to a nav menu item in the Home Icon menu, that image will be missing an `alt` attribute.

**Steps to Reproduce**

Inspect element of a custom icon `<img>` tag and verify the attr is not present

**Expected Behavior**

An empty alt attr (`alt=""`) should be present (to indicate that it is decorative).

**Possible Fix**

Add it.  https://github.com/UCF/Coronavirus-Child-Theme/blob/master/includes/nav-functions.php#L45

**Your environment:**

n/a

**Additional context**

We're getting flagged in SiteImprove for missing alt attrs.
